docs/spherical-video-v2-rfc.md

```diff
@@ -267,3 +267,155 @@ monoscopic equirectangular video:
               ...
 ```
 
+### WebM (Matroska)
+Spherical video metadata is stored in a new master element, `Projection`,
+placed inside a video track's `Video` master element.
+
+
+As the V2 specification stores its metadata in a different location, it is
+possible for a file to contain both the V1 and V2 metadata. If both V1 and
+V2 metadata are contained they should contain semantically equivalent
+information, with V2 taking priority when they differ.
+
+#### `Projection` master element
+##### Definition
+ID: 0x7670 \
+Level: 4 \
+Mandatory: No \
+Type: Master  \
+Default: N/A \
+Minver: 4 \
+WebM: Yes \
+Container: `Video` master element
+
+Describes the video projection details. Used to render spherical and VR videos.
+
+#### `ProjectionType` element
+##### Definition
+ID: 0x7671 \
+Level: 5 \
+Mandatory: Yes \
+Type: uinteger \
+Default: 0 \
+Minver: 4 \
+WebM: Yes \
+Container: `Projection` master element
+
+Describes the projection used for this video track.
+
+##### Semantics
+`ProjectionType` is an enum. The valid values are:
+
+* 0: Rectangular
+* 1: Equirectangular
+* 2: Cubemap
+
+
+#### `ProjectionPrivate` element
+##### Definition
+ID: 0x7672 \
+Level: 5 \
+Mandatory: No \
+Type: binary  \
+Default: N/A \
+Minver: 4 \
+WebM: Yes \
+Container: `Projection` master element
+
+Private data known only to the projection codec.
```

Comment:
nit: Not sure if "projection codec" is the best term here (or is this a WebM-ism?)